src/components/system/SystemAlert.jsx

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from "react";
+import { connect } from 'react-redux';
+import PropTypes from "prop-types";
+import * as R from 'ramda';
+import moment from 'moment';
+import { differenceInMinutes, differenceInHours } from 'date-fns';
+import systemActions from '../../features/system/actions';
+import '../../../assets/css/widgets.css';
+
+class Alert extends Component {
+
+  componentDidMount() {
+    this.getSystemInfo();
+  }
+
+  componentDidUpdate() {
+    this.getSystemInfo();
+  }
+
+  getSystemInfo() {
+    const { session, dispatch } = this.props;
+
+    if (!R.isEmpty(session) && session.authenticated) {
+      dispatch(systemActions.systemPollStartAction());
+    } else {
+      dispatch(systemActions.systemPollStopAction());
+    }
+  }
+
+  renderAlertMessage() {
+    const { system } = this.props;
+    const deviceTime = new Date();
+    const serverTime = new Date(`${system.systemDate}T${system.systemTime}Z`);
```

Comment:
@rhenshaw56 shouldn't this take into account the system.timeZone?

Reply:
the `system.timeZone` is still UTC, so I thought to just use that as it is all through

Reply:
but I don't think it's necessarily always in UTC...
that being said, you are on vacation, don't worry about it! :)